packages/voting/src/voting-calls.ts

```diff
@@ -877,10 +877,18 @@ export namespace VotingApi {
      * @param filters Optional criteria to filter the processes ID's given by the gateway
      * @param gateway
      */
-    export async function getProcessList(filters: { entityId?: string, namespace?: number, status?: VochainProcessStatus, withResults?: boolean, from?: number } = {}, gateway: IGatewayDVoteClient): Promise<string[]> {
+    export async function getProcessList(filters: { fromArchive: true, entityId?: string } | { fromArchive: false, entityId?: string, namespace?: number, status?: VochainProcessStatus, withResults?: boolean, from?: number }, gateway: IGatewayClient): Promise<string[]>
```

Comment:
```suggestion
    export async function getProcessList(filters: { fromArchive: true, entityId: string } | { fromArchive: false, entityId?: string, namespace?: number, status?: VochainProcessStatus, withResults?: boolean, from?: number }, gateway: IGatewayClient): Promise<string[]>
```

Reply:
It would be nice to have some sort of filtering... otherwise things may get unmanageable when entities start to have a lot of processes. Look at the case of Aragon Voice, the first page load needs to do a ton of requests